pytradfri/device/__init__.py

```diff
@@ -23,76 +26,88 @@ class Device(ApiResource):
     """Base class for devices."""
 
     @property
-    def application_type(self):
+    def application_type(self) -> TYPE_RAW:
         """Return application type."""
-        return self.raw.get(ATTR_APPLICATION_TYPE)
+        return cast(TYPE_RAW, self.raw.get(ATTR_APPLICATION_TYPE))
```

Comment:
`None`, which is returned if the key is missing in the dict, is not of `TYPE_RAW`.

Reply:
Corrected.